app/uk/gov/hmrc/exports/mongock/changesets/CacheChangeLog.scala

```diff
@@ -151,9 +162,27 @@ class CacheChangeLog {
       val setUpdateObj = new BasicDBObject(setUpdate)
 
       logger.info(s"Applying [5] changes to ${document.get("id")}")
-      db.getCollection(collection).findOneAndUpdate(objectToBeUpdated, setUpdateObj)
-      logger.info(s"Applied [5] changes to ${document.get("id")}")
+      getDeclarationsCollection(db).findOneAndUpdate(objectToBeUpdated, setUpdateObj)
     }
     logger.info("Applying 'CEDS-2247 Change routing countries structure... Done.")
   }
+
+  @ChangeSet(order = "006", id = "CEDS-2250 Add one structure level to /transport/borderModeOfTransportCode", author = "Maciej Rewera")
+  def updateTransportBorderModeOfTransportCode(db: MongoDatabase): Unit = {
+
+    logger.info("Applying 'CEDS-2250 Add one structure level to /transport/borderModeOfTransportCode'...")
+
+    getDeclarationsCollection(db).updateMany(
```

Comment:
Good! Whenever possible we need to be able to do these operations and delegate them to MongoDB
:+1: